### DISCUSSION TITLE
API indisponivel para download - Fundamentos do Blazor Web Assembly

### DISCUSSION THREAD

**Discussion by viniciusbernardo22** [balta-io/forum#492] [Tira Dúvidas] [answered]
Durante o módulo do curso de Blazor -> Fundamentos do Blazor Web Assembly é necessário o download de uma API para realizar as atividades, mas está indisponível. 

https://github.com/balta-io/2822/tree/main/src

Caminho:  src/encontro-premium-2023-05-04

**Comment by andrebaltieri** [comment, marked as answer]
https://github.com/balta-io/encontro-premium-2023-05-04
